doc/classes/File.xml

```diff
@@ -430,6 +431,7 @@
 			<description>
 				Stores the given [String] in the file.
 				Text will be encoded as UTF-8.
+				No newline is appended.
```

Comment:
I would recommend putting it on the first line instead of adding a 3rd paragraph. Also, please favor the direct voice. And it's an opportunity to improve the original docstring.

Suggestion:

> Appends [code]string[/code] to the file without a line return, encoding the text as UTF-8.